server/user/index.js

```diff
@@ -2,18 +2,18 @@
 
 const auth = require('passport').authenticate('jwt');
 const ctrl = require('./user.controller');
-const model = require('./user.model');
 const router = require('express-promise-router')();
 
 router
   // Public routes:
-  .post('/users/login', ctrl.login)
-  .post('/users/forgotPassword', ctrl.forgotPassword)
-  .post('/users/resetPassword', ctrl.resetPassword)
+  .post('/login', ctrl.login)
+  .post('/forgotPassword', ctrl.forgotPassword)
+  .post('/resetPassword', ctrl.resetPassword)
```

Comment:
Rename the route to use a [hyphen](https://restfulapi.net/resource-naming/) for multiword names instead of camel case.